docs/swagger-ui/swagger.json

```diff
@@ -636,11 +636,14 @@
           "200": {
             "description": "Success",
             "schema": {
-              "$ref": "#/definitions/NodeInfo"
+              "$ref": "#/definitions/MetadataChanges"
```

Comment:
This is incorrect, you'd need to change the `type` property in the `metadata` entry of `parameters`.
The returntype is unchanged.

Reply:
Oof, big mistake, indeed.

However, it seems to be impossible to use complex types (from https://swagger.io/docs/specification/v2_0/describing-parameters/#form-parameters):
> Form parameters are defined as in: formData. They can only be primitives (strings, numbers, booleans) or arrays of primitives (meaning you cannot use a $ref as the items value).

I will roll back this change, of course.

Reply:
Its not spec compliant, but the openapi-generator client gen doesnt fuss about it (with validation disabled).
It creates a method signature with `Object`, and then depending on the underlying http & json libs, it's might work or might not. The Spring libs seem to be able to handle it, the more generic clients not so much it seems.